test/ReadsUtils_server_test.py

```diff
@@ -680,6 +687,46 @@ def test_FASTQ_val_fail_bad_ext(self):
                             'File data/sample.txt is not a FASTQ file')
 
     # Upload tests ########################################################
+    def test_upload_fail_min_len_reads(self):
+
+        # The min_Sample.fastq file contains only one read with a length of 1.
+        read_length = 1
+
+        self.fail_upload_reads(
+            {'fwd_file': 'data/min_Sample.fastq',
+             'sequencing_tech': 'seqtech',
+             'wsname': self.ws_info[1],
+             'name': 'filereads1',
+             'min_read_length': read_length + 1},
+            "Invalid FASTQ file - Path: /kb/module/test/data/min_Sample.fastq."
+            )
+
+    def test_upload_min_len_reads(self):
+        tf = 'min_Sample.fastq'
+        target = os.path.join(self.scratch, tf)
+        shutil.copy('data/' + tf, target)
+        read_length = 1
+
+        ret = self.impl.upload_reads(
+            self.ctx, {'fwd_file': target,
+                       'sequencing_tech': 'seqtech',
+                       'wsname': self.ws_info[1],
+                       'name': 'filereads1'})
+        obj = self.dfu.get_objects(
+            {'object_refs': [self.ws_info[1] + '/filereads1']})['data'][0]
+        self.assertEqual(ret[0]['obj_ref'], self.make_ref(obj['info']))
+        self.assertEqual(obj['info'][2].startswith(
+            'KBaseFile.SingleEndLibrary'), True)
+        d = obj['data']
+        self.assertEqual(d['read_count'], read_length)
```

Comment:
Why would the number of reads be equal to the min read length parameter?

Reply:
👍 you are right. They have no relationship.

Reply:
:+1: